python/semantic_kernel/functions/kernel_function_from_prompt.py

```diff
@@ -175,11 +176,19 @@ async def _invoke_internal(self, context: FunctionInvocationContext) -> None:
                 kwargs["arguments"] = context.arguments
 
             try:
-                chat_message_contents = await prompt_render_result.ai_service.get_chat_message_contents(
-                    chat_history=chat_history,
-                    settings=prompt_render_result.execution_settings,
-                    **kwargs,
-                )
```

Comment:
Theoretically i forwarded the `invoke` to `get_chat_message` and i should not have break something, but since i do not have a huggingface and open ai key i could not verify it with the tests, once you approve the general idea i will clean this up.